components/org.wso2.carbon.identity.api.user.backupcode/org.wso2.carbon.identity.api.user.backupcode.v1/src/gen/java/org/wso2/carbon/identity/rest/api/user/backupcode/v1/MeApi.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.wso2.carbon.identity.rest.api.user.backupcode.v1;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.dto.*;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.MeApiService;
+import org.wso2.carbon.identity.rest.api.user.backupcode.v1.factories.MeApiServiceFactory;
+
+import io.swagger.annotations.ApiParam;
```

Comment:
Shall we re-arrange these imports in a methodical way?

Reply:
These are auto generated classes. AFAIK, we are not modifying these classes